features/bootstrap/FeatureContext.php

```diff
@@ -457,6 +457,24 @@ public function theOutputShouldContain(PyStringNode $text)
         );
     }
 
+    /**
+     * Checks whether last command output contains provided string including absolute paths
+     *
+     * @Then the output with absolute paths should contain:
+     *
+     * @param PyStringNode $text PyString text instance
+     */
+    public function theOutputWithAbsolutePathsShouldContain(PyStringNode $text)
```

Comment:
We add a new step that allows us to check the output without filtering out the absolute paths and instead replacing them with a placeholder